app/helpers/admin/users_helper.rb

```diff
@@ -16,4 +16,9 @@ def change_admin_attribute(user)
       content_tag(:button)
     end
   end
+
+  def role(userrole)
+    roles = ['User', 'Conf Admin', 'Admin']
+    roles[User.roles[userrole]]
+  end
```

Comment:
too much logic inside view. move as much as possible to view helper -> app/views/users/index.html.haml
try to create whole dropdown with a helper